olacodes/Project-Tracker
# Ft create dashboard

#### What does this PR do?
This PR creates a dashboard page and the component needed

#### List of task completed
- [x] Create dashboard page
- [x] Create ImageCard component
- [x] Create SearchBar component
- [x] Create SecondNav component
- [x] Create ProjectCard component

#### How can this be tested?
* click on netlify preview link below and add `/dashboard` to the route to see the dashboard page

Add more info if any. eg fixes #.
fix footer excess margin
